### PR TITLE
[UIIDM-58] Release v1.0.0

### PR DESCRIPTION
**Pull request by alb3rtino**
https://folio-org.atlassian.net/browse/UIIDM-58

Release v1.0.0

### CODE REVIEW

**Comment by alb3rtino** [comment]
whoops

File: CHANGELOG.md
Comment (elsenhans) [inline CHANGELOG.md:1]: Is there a reason why you do not add the link and release date like:
[10.0.0](https://github.com/folio-org/ui-erm-usage/tree/v10.0.0) (2024-10-29)

Comment (alb3rtino) [reply CHANGELOG.md:1]: I guess laziness.

Comment (elsenhans) [reply CHANGELOG.md:1]: It would be nice if we could always do it uniformly. It's not that much effort, after all.

**Pull request merged**